example_sns_init.yaml

```
@@ -112,9 +113,39 @@ Swap:
     restricted_countries:
         - AQ  # Antarctica
 
+    # Declare that each participant will get three SNS neurons in the following
+    # states:
+    # 1. Unlocked
+    # 2. Locked for 1 month
+    # 3. Locked for 2 months
     VestingSchedule:
-        events: 104
-        interval: 7 d
+        events: 3
+        interval: 1 month
+
+    # Absolute time within a day when the swap is supposed to start.
+    #
+    # An algorithm will be applied to allow at least 24 hours between the time
+    # of execution of the CreateServiceNervousSystem proposal and swap start.
+    # For example, if start_time is 23:30 UTC and the proposal is adopted and
+    # executed at 23:20 UTC, then the swap start will be at 23:30 UTC the next
+    # day (i.e., in 24 hours and 10 min from the proposal execution time).
+    #
+    # WARNING: Swap start_time works differently on mainnet and in testing.
+    #
+    # On mainnet:
+    # - Setting start_time to some value (e.g., 12:00 UTC) will allow the swap
```

```suggestion
    # - Setting start_time to some value (e.g., 23:30 UTC) will allow the swap
```
let's reuse the same timestamp as in the example above